plugin/src/main/java/org/opensearch/ml/action/model_group/TransportRegisterModelGroupAction.java

```diff
@@ -153,14 +153,19 @@ public void createModelGroup(MLRegisterModelGroupInput input, ActionListener<Str
     }
 
     private void validateRequestForAccessControl(MLRegisterModelGroupInput input, User user) {
-        ModelAccessIdentifier modelAccessIdentifier = input.getModelAccessIdentifier();
+        ModelAccessMode modelAccessMode = input.getModelAccessMode();
         Boolean isAddAllBackendRoles = input.getIsAddAllBackendRoles();
-        if (modelAccessIdentifier == null) {
-            throw new IllegalArgumentException("Invalid model access identifier");
-        } else if ((ModelAccessIdentifier.PUBLIC == modelAccessIdentifier || ModelAccessIdentifier.PRIVATE == modelAccessIdentifier)
+        if (modelAccessMode == null) {
+            if (!Boolean.TRUE.equals(isAddAllBackendRoles) && CollectionUtils.isEmpty(input.getBackendRoles())) {
+                throw new IllegalArgumentException("User must specify at least one backend role or make the model public/private");
```

Comment:
If we do not specify any access fields, the default should be private actually, not throw exception

Reply:
I think let user aware of this need more time, explicitly specify the model mode to private would be easy to understand and less confuse to user, let's make this explicit, if there's callout from user, we can change the behavior.

Reply:
yeah should be fine then